app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,23 @@
+!!!
+%html
+  %head
+    %meta{content: "text/html; charset=UTF-8", "http-equiv": "Content-Type"}/
+    %title App
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': true
+    = javascript_include_tag 'application', 'data-turbolinks-track': true
+    = csrf_meta_tags
+  %body
+    %nav.navbar.navbar-static-top.navbar-inverse
+      .container
+        %nav#navbar.collapse.navbar-collapse
+          %a.navbar-brand{href: "/"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace